serverless.yml

```diff
@@ -69,6 +69,11 @@ functions:
     events:
       - websocket:
           route: unfollow
+  has-thread:
+    handler: src/threads/hasThread.handler
+    events:
+      - websocket:
+          route: has-thread
```

Comment:
probably it should be called "thread-status" if it returns more than a boolean

Reply:
fixed